tools/testfiles/tfloatsattrs.ddl

```diff
@@ -0,0 +1,621 @@
+HDF5 "tfloatsattrs.h5" {
+GROUP "/" {
+   DATASET "DS128BITS" {
+      DATATYPE  128-bit little-endian floating-point
```

Comment:
"128-bit little-endian floating-point" is not consistent with all the other datatype output, which uses HDF5's pre-defined datatype IDs, e.g. "H5T_IEEE_F32LE" and "H5T_IEEE_F64LE" (see lower in this file's output)